.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,8 +3,8 @@ quickly and easily:
 
  - [ ] Make sure the PR title is formatted like:
    `[BEAM-<Jira issue #>] Description of pull request`
- - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
-       Travis-CI on your fork and ensure the whole test matrix passes).
+ - [ ] Make sure tests pass via `mvn clean install -Pall-checks`. (Even better,
```

Comment:
I actually don't think we should add this. I want the user to immediately open a PR when it compiles. Running `mvn test` is a bonus. Then Jenkins & Travis can do the thorough stuff. If there is a need to iterate, the reviewer should advise on the invocation with `-P all-checks`.